diff -up spec-helper-0.31.49/Makefile.omv~ spec-helper-0.31.49/Makefile
--- spec-helper-0.31.49/Makefile.omv~	2020-03-30 16:52:22.683743966 +0200
+++ spec-helper-0.31.49/Makefile	2020-03-30 16:52:27.068757270 +0200
@@ -4,8 +4,7 @@ SVNPATH = git@github.com:OpenMandrivaSof
 
 SCRIPT_FILES =  clean_files clean_perl check_elf_files \
 		lib_symlinks fix_file_permissions fix_mo fix_xdg fix_pkgconfig fix_pamd \
-		remove_info_dir remove_libtool_files remove_rpath relink_symlinks fix_eol \
-		python_shebangs
+		remove_info_dir remove_libtool_files remove_rpath relink_symlinks fix_eol
 BIN_FILES    = rediff_patch spec-cleaner
 MACROS_FILES = spec-helper.macros
 TEST_FILES   = t/*.t
diff -up spec-helper-0.31.49/spec-helper.macros.in.omv~ spec-helper-0.31.49/spec-helper.macros.in
--- spec-helper-0.31.49/spec-helper.macros.in.omv~	2020-03-30 16:52:31.624771086 +0200
+++ spec-helper-0.31.49/spec-helper.macros.in	2020-03-30 16:52:37.536789004 +0200
@@ -16,7 +16,6 @@
     %{?!dont_fix_eol:			[ -n "$DONT_FIX_EOL" ]		|| %_spec_helper_dir/fix_eol} \
     %{?!dont_check_elf_files:		[ -n "$DONT_CHECK_ELF_FILES" ]	|| %_spec_helper_dir/check_elf_files} \
     %{?!dont_remove_rpath:		[ -n "$DONT_REMOVE_RPATH" ]	|| %_spec_helper_dir/remove_rpath} \
-    %{?!dont_check_python_shebangs:		[ -n "$DONT_CHECK_PYTHON_SHEBANGS" ]	|| %_spec_helper_dir/python_shebangs} \
     %{?!__debug_package:		[ -n "%{?dont_strip:1}%{?!dont_strip:$DONT_STRIP}" ] || export DISABLE_DEBUG=1 && %__debug_install_post} \
 %{nil}
 
